substrate/frame/vesting/src/benchmarking.rs

```diff
@@ -383,6 +383,34 @@ benchmarks! {
 		);
 	}
 
+force_remove_vesting_schedule {
+		let l in 0 .. MaxLocksOf::<T>::get() - 1;
+		let s in 2 .. T::MAX_VESTING_SCHEDULES - 1;
+
+		let source: T::AccountId = account("source", 0, SEED);
+		let source_lookup: <T::Lookup as StaticLookup>::Source = T::Lookup::unlookup(source.clone());
+		T::Currency::make_free_balance_be(&source, T::Currency::minimum_balance());
+
+		let target: T::AccountId = account("target", 0, SEED);
+		let target_lookup: <T::Lookup as StaticLookup>::Source = T::Lookup::unlookup(target.clone());
+
+		// Give target existing locks.
+		add_locks::<T>(&target, l as u8);
+		let _ = add_vesting_schedules::<T>(target_lookup.clone(), s)?;
+
+		let transfer_amount = T::MinVestedTransfer::get();
```

Comment:
This line doesn't seem to serve any purpose.

Reply:
When this is refactored to v2, compiler would alert for simple things like this. Inside the macro, compiler is quiet. So I'll work on the v2 refactor after this is merged, it's much easier to reason about the benchmark tests in the v2 structure.